### DISCUSSION TITLE
Can I change the save game directory?  (...and where is it?)

### DISCUSSION THREAD

**Discussion by sofakng** [Novum/vkQuake#609] [Q&A]
Is it possible to specify a directory (or network location) for the save games?

I'd like to use a network share so I can share save games between multiple PCs on my network.

**Comment by Novum** [comment]
If you use the installer the save directory is in the user directory (%APPDATA%), otherwise it's in the game directory in the mod folder (id1 for the base game)

**Reply by sofakng** [reply]
Thanks... I'm sorry but I should have mentioned I'm using macOS on one of my machines.

Can I specify the mod folder directory (ie. 'id1') with a full path on the command line?

**Comment by Novum** [comment]
You can specify the entire game folder with `-basedir`

**Reply by sofakng** [reply]
Great, thanks so much!

**Reply by sofakng** [reply]
This is working, but it also appears to save the vkquake.cfg into that directory.  Is that intended?

I'm trying to have several different computers (with different video resolutions, etc) share the same game/save data so each computer would have it's own vkquake.cfg but share the id1 (and other) folders.

**Reply by Novum** [reply]
Does any other game offer this? Seems very obscure.

**Reply by sofakng** [reply]
Yeah, it's a bit obscure but I'm not sure of a better way to share save games between multiple computers (with different video resolution, operation system [macOS/Windows/SteamOS], etc).

Anyways, Crispy Doom lets you specify the WAD on the command-line (-iwad) and the save directory (-savedir) but saves it's configuration file on the local machine which works the best in my situation.

I'm really just looking for a way to share the save games.  If vkQuake had a -savedir parameter (but required the game data/config on the local machine) that would also work.

**Reply by sto6** [reply]
necro-thread but anyway, observations that might yield a (convoluted) solution:

1. for me on macos, vkQuake.cfg can be made **readonly**, and it won't be overwritten.
2. From any given .cfg, isn't there a command like **exec** or **source** by which to load another .cfg file by name?

Then once your platform specific vkQuake.cfg's are stable, perhaps they can be a readonly one-liner to load the "real" file which is actually a symlink (macos or linux, problematic on Windows?) to a system-local file (i.e. outside of the network-disk).

Also if there is an exec or source command, your launcher on each platform could include cmd-line args to do it: "+exec /tmp/my.cfg" (repeat as needed for additional files), there-by loading system-specific configs.
